### PR TITLE
Initial analytics opt-out page for internal users

### PR DESCRIPTION
**Pull request by robertfairhead**
Related to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3624

Will set a user-scoped custom dimension in Google Analytics that we can add a filter for to drop our internal user traffic in production. This is an opt-in filter and will need to be set for each computer/browser combination but should endure across sessions.

### CODE REVIEW

**Comment by robertfairhead** [comment]
@ayaleloehr @suzchap : This is the initial pass at the analytics opt-out for vets.gov team members in production. You can see the site at: https://vetsgov-pr-6106.herokuapp.com/analytics-opt-out.html if you'd like to review the layout/copy

I set it so that it does not appear in our sitemap.xml and is disallowed in robot.txt to prevent it coming up in search results.

@robbiethegeek-usds added you for review as a general website reviewer since this is non-application specific

Once this is in production, can do some testing to ensure the custom dimension is working properly and then proceed with next steps listed in the referenced issue above.

File: content/pages/analytics-opt-out.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<!-- Maintenance Page Start -->
+
+<div class="main home" role="main">
+  <div class="section main-menu">
+    <div class="row">
+      <div class="small-12 columns">
+        <div style="padding: 2em 0;">
+        <h3>Analytics Opt-out</h3>
+        <h4>Click the below button to opt out of the vets.gov Google Analytics collection.</h4>
+        <p>This opt-out is durable for this computer/browser combination. It is stored in cookie. To re-enable vets.gov Google Analytics collection, you will need to clear your cookies.</p>
```
Comment (ayaleloehr) [inline content/pages/analytics-opt-out.md]: Nit: "It is stored in cookie" feels off. Maybe "It is stored in a cookie"?

File: content/pages/analytics-opt-out.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        <p>The intended use case of this opt-out is for vets.gov team members performing testing or validation in the production environment. Other uses are not recommended.</p>
```
Comment (ayaleloehr) [inline content/pages/analytics-opt-out.md]: Nit: Capital V on Vets.gov

**Review by ayaleloehr — APPROVED** [review]
Two pedantic nits, but LGTM.

**Pull request merged**